app/src/organisms/InstrumentMountItem/ProtocolInstrumentMountItem.tsx

```diff
@@ -61,34 +63,26 @@ export function ProtocolInstrumentMountItem(
   props: ProtocolInstrumentMountItemProps
 ): JSX.Element {
   const { t, i18n } = useTranslation('protocol_setup')
-  const { mount, attachedInstrument, speccedName } = props
+  const { mount, attachedInstrument, speccedName, mostRecentAnalysis } = props
 
-  const [showChoosePipetteModal, setShowChoosePipetteModal] = React.useState(
+  const [showPipetteWizardFlow, setShowPipetteWizardFlow] = React.useState(
     false
   )
-  const [
-    selectedPipette,
-    setSelectedPipette,
-  ] = React.useState<SelectablePipettes>(SINGLE_MOUNT_PIPETTES)
+  const [flowType, setFlowType] = React.useState('ATTACH')
```

Comment:
Maybe we need to make a new `flowType` type for this component to use.

```suggestion
  const [flowType, setFlowType] = React.useState<FlowType>('ATTACH')
```